### PR TITLE
partially solves issue #422

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: src/clients/AdvancedTab.tsx [hunk 1]
```diff
@@ -178,96 +176,95 @@ export const AdvancedTab = ({
...
-              labelIcon={
-                <HelpItem
-                  helpText="clients-help:notBefore"
-                  forLabel={t("notBefore")}
-                  forID="kc-not-before"
-                />
-              }
+      <>
+        <Text className="pf-u-py-lg">
+          <Trans i18nKey="clients-help:notBeforeIntro">
```
Comment (mfrances17) [inline src/clients/AdvancedTab.tsx:181]: should use the "t" function here

**Review by mfrances17 — APPROVED** [review]
LGTM as a partial solve for the issues in 422

File: src/clients/AdvancedTab.tsx (hunk 1 above)
Comment (edewit) [reply src/clients/AdvancedTab.tsx:181]: I can't use the `t` function as I need a component in the translated text

Comment (mfrances17) [reply src/clients/AdvancedTab.tsx:182]: ah, ok

**Pull request merged**